napari/__main__.py

```diff
@@ -177,22 +178,23 @@ def _run():
                 'When providing a python script, only a '
                 'single positional argument may be provided'
             )
+        runpy.run_path(args.paths[0])
 
-        with gui_qt(startup_logo=True) as app:
-            if hasattr(app, '_splash_widget'):
-                app._splash_widget.close()
-            runpy.run_path(args.paths[0])
-            if getattr(app, '_existed', False):
-                sys.exit()
     else:
-        with gui_qt(startup_logo=True, gui_exceptions=True):
-            view_path(
-                args.paths,
-                stack=args.stack,
-                plugin=args.plugin,
-                layer_type=args.layer_type,
-                **kwargs,
-            )
+
+        from ._qt.widgets.qt_splash_screen import NapariSplashScreen
```

Comment:
Do you want to use the absolute import here too? You use it above to import `napari`, I guess not critical either way, but curious if intentional or not to use relative here and absolute above

Reply:
not intentional to use them differently.  I did use `napari` directly everywhere else though as it's _much_ easier to monkey patch in tests (and I personally prefer absolute imports in a  `__main__.py`).  will change here too

Reply:
yeah, let's go absolute import here too